keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -331,7 +331,7 @@ metadata:
   serviceAccountName: [5]
   accessSecretName: [6]
   provisioner: [7]
-  policyActions [8]
+  policyActionsConfigMapRef: [8]
```

Comment:
This is not a ref to a CM. Instead it is a string containing the base-64 encoded CM.data k-v pairs.